"""Bug fix #113631741

Revision ID: efa3e282e9c4
Revises: 36316acef1a
Create Date: 2016-02-12 16:23:47.953295

"""

# revision identifiers, used by Alembic.
revision = 'efa3e282e9c4'
down_revision = '36316acef1a'

from alembic import op
import sqlalchemy as sa
from sqlalchemy.dialects import postgresql

def upgrade():
    ### commands auto generated by Alembic - please adjust! ###
    op.drop_table('user_search_and_results')
    ### end Alembic commands ###


def downgrade():
    ### commands auto generated by Alembic - please adjust! ###
    op.create_table('user_search_and_results',
    sa.Column('viewed_time', postgresql.TIMESTAMP(timezone=True), autoincrement=False, nullable=False),
    sa.Column('user_id', sa.VARCHAR(length=20), autoincrement=False, nullable=False),
    sa.Column('title_number', sa.VARCHAR(length=20), autoincrement=False, nullable=False),
    sa.Column('search_type', sa.VARCHAR(length=20), autoincrement=False, nullable=False),
    sa.Column('purchase_type', sa.VARCHAR(length=1), autoincrement=False, nullable=False),
    sa.Column('amount', sa.VARCHAR(length=10), autoincrement=False, nullable=False),
    sa.Column('cart_id', sa.VARCHAR(length=30), autoincrement=False, nullable=True),
    sa.Column('transaction_id', sa.VARCHAR(length=30), autoincrement=False, nullable=True),
    sa.PrimaryKeyConstraint('viewed_time', 'user_id', name='user_search_and_results_pkey')
    )
    ### end Alembic commands ###
